cmd/edit/addon/cmd.go

```diff
@@ -42,11 +42,9 @@ var Cmd = &cobra.Command{
 	Run:                run,
 	DisableFlagParsing: true,
 	Args: func(cmd *cobra.Command, argv []string) error {
-		err := cmd.ParseFlags(argv)
+
+		err := arguments.PreprocessUnknownFlagsWithId(cmd, argv)
```

Comment:
@hunterkepley We're discarding the `err` returned from your function. Is there anyway we can wrap it so that we can more easily debug failures? It also will help the user fix any problems with the way they are using the command.